course/meetings/eeb-asher-cutter-20170303.md

```diff
@@ -0,0 +1,111 @@
+---
+Contact: Asher Cutter
+Department: Ecology and Evolutinoary Biology
+Uoftcoders: Madeleine, Luke, Lina, Joel
+Date: 20170303
```

Comment:
Not sure how much this matters, but generally the fields in the YAML header are all small caps (`contact`, `department`)